doc/source/options.rst

```diff
@@ -454,14 +456,14 @@ For instance:
 
    pd.set_eng_float_format(accuracy=3, use_eng_prefix=True)
    s = pd.Series(np.random.randn(5), index=['a', 'b', 'c', 'd', 'e'])
-   s/1.e3
-   s/1.e6
+   s / 1.e3
+   s / 1.e6
 
 .. ipython:: python
    :suppress:
    :okwarning:
 
-   pd.reset_option('^display\.')
+   pd.reset_option(r'^display.')
```

Comment:
I don't think we want to delete the backslash, I guess this this string is used in a regular expression